maquetado/scss/_tech.scss

```diff
@@ -19,32 +19,21 @@
 
 .tech {
   align-items: center;
-  background-color: $green;
-  border-radius: 5px;
   display: flex;
   flex-direction: column;
-  margin: 10px;
-  max-width: 100%;
+  margin: auto 0;
```

Comment:
Delete this, isn't necessary. You can replace this in `.section-tech` for `grid-gap`